carta/cpp/core/ScriptedClient/ScriptFacade.cpp

```diff
@@ -768,10 +772,15 @@ QStringList ScriptFacade::getImageDimensions( const QString& controlId ) {
     if ( obj != nullptr ){
         Carta::Data::Controller* controller = dynamic_cast<Carta::Data::Controller*>(obj);
         if ( controller != nullptr ){
-            resultList = controller->getImageDimensions( );
-            if ( resultList[0] == "" ) {
+            std::vector<int> dimensions = controller->getImageDimensions();
+            if ( dimensions.size() == 1 && dimensions[0] == 0 ) {
```

Comment:
Hmmm, I wonder why the error case would not be an empty list, it would certainly require less code to check for it:)

I don't know if there are 'images' out there that have at least one dimension set to zero. They certainly could not contain any data, but maybe such images could exist as containers of meta-data only? Maybe @low-sky would know?

In any case, I don't have any more comments regarding this code now that I know [0] is the only special error case returned by getImageDimensions().